packages/e2e-test-utils/src/get-block-title.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a string containing the block title associated with the provided block name.
+ * @param {string} blockName Block name.
+ *
+ * @return {Promise} Promise resolving with a string containing the block title.
+ */
+export async function getBlockTitle( blockName ) {
```

Comment:
How about making it more generic: `getBlockSetting( blockName, settingName )`?

Another idea, what if we create a wrapper over data API? It might not scale to keep creating helpers each time we want to get some data from the store.